Classes/Emogrifier.php

```diff
@@ -404,6 +404,23 @@ protected function process(\DOMDocument $xmlDocument)
         $this->copyCssWithMediaToStyleNode($xmlDocument, $xPath, $cssParts['media']);
     }
 
+
+    /**
+     * Applies $this->css to $xmlDocument and handle process errors.
+     *
+     * This method places the CSS inline.
+     *
+     * @param \DOMDocument $xmlDocument
+     *
+     * @return void
+     */
+    private function processWithErrorHandling(\DOMDocument $xmlDocument)
```

Comment:
Thanks for moving this to a separate method.

process now only is called from this method. So I think the error handling could go directly into process. What do you think?